keras_contrib/layers/advanced_activations/pseu.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+from keras import initializers
```

Comment:
Usually, we like to have the imports sorted alphabetically, with first all the `import XXX` and then all the `from XX import YY`. See [this StackOverflow thread](https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement)

Reply:
Fixed